stdlib/lazy.mli

```diff
@@ -67,6 +68,23 @@ external force : 'a t -> 'a = "%lazy_force"
    recursively.
 *)
 
+val is_val : 'a t -> bool
```

Comment:
While you are improving the interface, what about removing that spurious `(* val force : 'a t-> 'a *)` above?